test/api.js

```diff
@@ -314,9 +314,14 @@ test('test file in node_modules is ignored', function (t) {
 test('Node.js-style --require CLI argument', function (t) {
 	t.plan(1);
 
+	var requirePath = path.relative(process.cwd(), path.join(__dirname, 'fixture', 'install-global.js'));
+	// `path` will use \\ to seperate paths on windows, but require doesn't.
+	requirePath = requirePath.split(path.sep).join('/');
```

Comment:
@jamestalmage: it's iffy, because node doesn't allow that (AFAIK), and 'require' doesn't allow that, so it would be a strange thing to do, but I'll add it if you really want.